docs/README-dev.md

```diff
@@ -58,11 +58,15 @@ Because of the above, include paths in source should always feature the full pat
 Note that the recommended way of building the project is to use separate build directory, where CMake will generate all targets. In this way, you can keep a clean source directory. The following shell commands illustrate this usage:
 
 ```sh
-  cd <colobot_repository>
+  git clone https://github.com/colobot/colobot.git
+  cd colobot
+  git submodule update --init
```

Comment:
61 and 63 could be simplified as
`git clone --recursive https://github.com/colobot/colobot.git`

Reply:
Done